scripts/wmt/prepare_data.sh

```diff
@@ -19,20 +21,54 @@
 # Tests sets: http://data.statmt.org/wmt17/translation-task/test.tgz
 ##################################################################################
 
+# provide script usage instructions
 if [ $# -eq 0 ]
 then
     echo "usage: $0 <data_dir>"
     exit 1
 fi
 
+# set relevant paths
 SP_PATH=/usr/local/bin
 DATA_PATH=$1
 TEST_PATH=$DATA_PATH/test
 
+# set vocabulary size and source and target languages
 vocab_size=32000
 sl=en
 tl=de
 
+# Download the default datasets into the $DATA_PATH; mkdir if it doesn't exist
+mkdir -p $DATA_PATH
+cd $DATA_PATH
+#
+# echo "Downloading and extracting Commoncrawl data (919 MB) for training..."
```

Comment:
I think you should uncomment this download as it is uncommented in the `corpus` set.

Reply:
Yes, forgot about uncommenting these lines, thanks for spotting!